README.md

```diff
@@ -7,18 +7,24 @@
 </div>
 
 ## Overview
```

Comment:
Let's revert changes to the TOC, I intentionally didn't include every item for a better overview.

Reply:
Done